### PR TITLE
Add merge functionality

### PR DESCRIPTION
**Pull request by stefanste**

### CODE REVIEW

**Comment by stefanste** [comment]
Comments addressed

File: spec/synchronizer_spec.rb [hunk 1]
```diff
@@ -51,11 +52,17 @@ def create_remote_file(path, body)
     connection.put_object(bucket_name, remote_directory.join(path).to_s, body)
   end
 
+  def create_local_file(file, content)
+    File.open(local_directory.join(file), 'w') do |f|
```
Comment (adz) [inline spec/synchronizer_spec.rb]:
``` ruby
File.write(local_directory.join(file), content)
```

**Comment by stefanste** [comment]
:+1:

**Comment by mnoack** [comment]
:+1:

**Pull request merged**